src/wrap_cl.hpp

```diff
@@ -2574,6 +2574,55 @@ namespace pyopencl
     PYOPENCL_RETURN_NEW_EVENT(evt);
   }
 
+#ifdef CL_DEVICE_P2P_DEVICES_AMD
```

Comment:
Remove this?

Reply:
Not sure about that, because some of the other extensions that are defined in [pyopencl_ext.h](https://github.com/inducer/pyopencl/pull/697/commits/d725cb7a4c6ff71a48f0e8f6abce36da0689b8db#diff-36495b4bd7a883054c0b55862341cd3a4da2b8f25cf8a3aacc5f94aac9ebf5e6) have a corresponding `#ifdef` as well.